validator/src/main/java/com/amazon/aoc/validators/ConatinerInsightECSStructuredLogValidator.java

```diff
@@ -0,0 +1,69 @@
+package com.amazon.aoc.validators;
+
+import com.amazon.aoc.helpers.MustacheHelper;
+import com.amazon.aoc.models.Context;
+import com.amazon.aoc.models.JsonSchemaFileConfig;
+import com.amazonaws.services.logs.model.FilteredLogEvent;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.io.FilenameUtils;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+
+@Log4j2
+public class ConatinerInsightECSStructuredLogValidator
+        extends AbstractStructuredLogValidator {
+
+  private static final List<String> LOG_TYPE_TO_VALIDATE = Arrays.asList(
+          "Instance",
+          "InstanceDiskIO",
+          "InstanceFS",
+          "InstanceNet"
+  );
+
+  private static final int MAX_RETRY_COUNT = 15;
+  private static final int QUERY_LIMIT = 100;
+
+  @Override
+  void init(Context context, String templatePath) throws Exception {
+    logGroupName = String.format("/aws/containerinsights/%s/performance",
```

Comment:
can we define these magic code as constants?

Reply:
Is this log group name correct? I think ecs container insight has `ecs` in log group name, they one w/o `ecs` is for `eks`, you can deploy a cwagent and see it's behaviour.  Also the log group is used by ecsagent (or other agent?) as well for generating most metrics (except for instance).

Though even if the logroup name is wrong, as long as emf log has the right namespace, container insight dashboard still work (at least for prometheus, didn't try ecs instance).

Reply:
@pingleig  Yea, you are right, the log group name should be` /aws/ecs/containerinsights/{ClusterName}/performance`